mmpose/datasets/datasets/hand/panoptic_dataset.py

```diff
@@ -1,18 +1,14 @@
 import os
 from collections import OrderedDict
 
-import json_tricks as json
 import numpy as np
-from xtcocotools.coco import COCO
 
-from mmpose.core.evaluation.top_down_eval import (keypoint_auc, keypoint_epe,
-                                                  keypoint_pck_accuracy)
 from mmpose.datasets.builder import DATASETS
-from .topdown_base_dataset import TopDownBaseDataset
+from .hand_base_dataset import HandBaseDataset
 
 
 @DATASETS.register_module()
-class TopDownPanopticDataset(TopDownBaseDataset):
+class PanopticDataset(HandBaseDataset):
```

Comment:
One way to help migration is add a dummy class in the original file:

```python
@DATASETS.register_module()
class TopDownPanopticDataset(TopDownBaseDataset):
    def __init__(self, *args，**kwargs):
        raise(ImportError('TopDownPanopticDataset has been renamed into PanopticDataset'))
```

This way, users will know how to change their code